app/actions/websocket/teams.ts

```
@@ -0,0 +1,37 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {DeviceEventEmitter} from 'react-native';
+
+import {localRemoveUserFromTeam} from '@actions/local/team';
+import {updateUsersNoLongerVisible} from '@actions/remote/user';
+import {queryCurrentTeamId} from '@app/queries/servers/system';
+import {queryCurrentUser} from '@app/queries/servers/user';
+import {isGuest} from '@app/utils/user';
+import DatabaseManager from '@database/manager';
+
+export async function handleLeaveTeamEvent(serverUrl: string, msg: any) {
+    const database = DatabaseManager.serverDatabases[serverUrl];
+    if (!database) {
+        return;
```

should we throw an error here as we do elsewhere?